### ISSUE TITLE
Couldn't use this project on React Native 0.40

### ISSUE THREAD

**Issue by guilhermebruzzi** [nparashuram/maya-kai#5]
An error ocurred when I tried to run the last version of this project with react-native 0.40:

Unable to resolve module EventPluginRegistry from <path_to_my_project>/node_modules/maya-kai/src/inde.js Modyle does not exist in the module map or in these directies:  <path_to_my_project>/node_modules

To temporarily fix it, I had to manually edit node_modules/maya-kai/src/index.js to import the libs from a relative path:

import EventPluginRegistry from '../../react-native/Libraries/Renderer/src/renderers/shared/stack/event/EventPluginRegistry.js';

import ReactNativeEventEmitter from '../../react-native/Libraries/Renderer/src/renderers/native/ReactNativeEventEmitter.js';

**Comment by axemclion** [comment]
I think i fixed it in this commit - https://github.com/nparashuram/maya-kai/commit/08bc572034b0953698d5115a8ea0f9474f19e2cd

Is that not working for you ?

**Comment by guilhermebruzzi** [comment]
It isn't. And when I interact inside a webview it doesn't know how to replay the steps. To make it work on a webview, I should use browserSync?

**Comment by axemclion** [comment]
For a webview, you would have to use browsersync. This package basically uses React Native's event model. The Webview does not send user actions to taht.

**Comment by axemclion** [comment]
I am thinking of adding

```
 import EventPluginRegistry from 'react-native/Libraries/Renderer/src/renderers/shared/stack/event/EventPluginRegistry.js';

import ReactNativeEventEmitter from 'react-native/Libraries/Renderer/src/renderers/native/ReactNativeEventEmitter.js';

```

**Comment by axemclion** [comment]
@guilhermebruzzi Fixed it in npm @ 0.0.7
